pkg/kube/translation/annotations/redirect.go

```diff
@@ -15,11 +15,15 @@
 package annotations
 
 import (
+	"strconv"
+
 	apisixv1 "github.com/apache/apisix-ingress-controller/pkg/types/apisix/v1"
 )
 
 const (
 	_httpToHttps = AnnotationsPrefix + "http-to-https"
+	_uri         = AnnotationsPrefix + "uri"
+	_retCode     = AnnotationsPrefix + "ret_code"
```

Comment:
I tend to use

`k8s.apisix.apache.org/permanent-redirect` for uri
`k8s.apisix.apache.org/permanent-redirect-code` for ret_code

Reply:
got it, I think so, too.